include/deal.II/grid/grid_tools.h

```diff
@@ -230,6 +230,43 @@ namespace GridTools
   double
   cell_measure(const T &, ...);
 
+  /**
+   * Computes an aspect ratio measure for all cells and fills a vector with one
```

Comment:
Just write: `Computes an aspect ratio measure for all locally owned active cells...`. That should please @tjhei!?

Reply:
yes, that is better.